x11-misc/clipmenu: initial commit

ferki opened the pull request:
This PR adds x11-misc/clipmenu-6.0.1 to the portage tree, along with x11-misc/clipnotify-1.0.2 as dependency.

I'd also would like to step up as proxied maintainer for these packages.

Please review, then either merge or let me know if something needs to be modified first.

Closes: https://bugs.gentoo.org/722228

DarthGandalf commented on x11-misc/clipnotify/clipnotify-1.0.2.ebuild:
```diff
@@ -0,0 +1,24 @@
+	x11-libs/libXfixes
+"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
```
DEPEND and BDEPEND packages are built for different architectures when cross-compiling

ferki replied:
@DarthGandalf: I'm not entirely sure what's the expected action in response to that remark from my side. What would you suggest using instead that would be acceptable?

DarthGandalf replied:
Please review https://devmanual.gentoo.org/general-concepts/dependencies/index.html and decide what dependency goes to where. Perhaps, by removing BDEPEND if the build system doesn't need anything installed.

ferki commented:
I've made the following changes:

- rebased on top of current master
- removed BDEPEND from x11-misc/clipnotify
- introduced a local variable for the install loop (thanks @jonasstein!)
- bumped x11-misc/clipmenu to latest version (6.1.0), which was just released meanwhile
